### PR TITLE
Config GUI

### PR DESCRIPTION
**Pull request by StartForKillerMC**
I made a config gui with ModMenu, and added support with I18n.

### CODE REVIEW

**Comment by StartForKillerMC** [comment]
It may need some translations that original files dont have.

**Comment by mikroskeem** [comment]
Will try it out today and merge.

File: src/main/java/eu/mikroskeem/worldeditcui/gui/SettingsEntry.java [hunk 1]
```diff
@@ -29,11 +29,23 @@ public void render(int index, int y, int x, int rowWidth, int rowHeight, int mou
...
+            for(String text : name.split(" ")) {
+                if((text.length() + lengthTemp) <= trimmedName.length()) {
+                    lengthTemp += text.length();
+                    temp1String += text + " ";
+                } else {
+                    temp2String += text + " ";
+                }
+            }
+            this.client.textRenderer.draw(temp1String, x + 50, y - 4, 0xFFFFFF);
+            this.client.textRenderer.draw(temp2String, x + 50, y + 6, 0xFFFFFF);
```
Comment (mikroskeem) [inline src/main/java/eu/mikroskeem/worldeditcui/gui/SettingsEntry.java:45]: What does this section do, split a long line up to two lines? Can't this be implemented in a better way? I see a possibility to use StringBuilder here.

Comment (StartForKillerMC) [reply src/main/java/eu/mikroskeem/worldeditcui/gui/SettingsEntry.java:45]: Yeah, its a section that maybe is too messy, it split the string if its too long, depending on the acreen width

**Comment by StartForKillerMC** [comment]
This should be better

**Comment by mikroskeem** [comment]
image

Line wrapping is messed up a bit, could you try giving it a new shot?

**Comment by StartForKillerMC** [comment]
Yeah, i can rework it.

**Comment by StartForKillerMC** [comment]
This should work better.

**Comment by mikroskeem** [comment]
image

Looks good

**Comment by mikroskeem** [comment]
Thank you.

**Pull request merged**